3-extra/1-factorial.js

```diff
@@ -9,7 +9,11 @@
 */
 
 function factorial(input) {
-    // TODO
+     let sum = 1
```

Comment:
Very small detail, but this is not a sum but a product. The function is correct though.